### DISCUSSION TITLE
Version v1.1.1

### DISCUSSION THREAD

**Discussion by AIGamesDeveloper** [Sprout-Inc/Sprout-Scratch#4] [Announcements]
v1.1.1 has added more features such as 

* A Ban list
* An appeal link
* More moderators

any feedback is appreciated!

<hr /><em>This discussion was created from the release <a href='https://github.com/Sprout-Inc/Sprout-Scratch/releases/tag/v1.1.1'>Version v1.1.1</a>.</em>

**Comment by Kerbalistic-coder** [comment]
make the ban list have any thing with brode get banned
